src/main/java/kr/codesquad/ladder/domain/LadderGame.java

```diff
@@ -0,0 +1,32 @@
+package kr.codesquad.ladder.domain;
+
+import kr.codesquad.ladder.view.InputView;
+import kr.codesquad.ladder.view.OutputView;
+
+import java.io.IOException;
+import java.util.List;
+
+public class LadderGame {
+    private Ladder ladder;
+    private InputView inputView;
+    private OutputView outputView;
+
+    public LadderGame() {
+        this.inputView = new InputView();
+        this.outputView = new OutputView();
+    }
+
+    public void run() {
+        try {
+            List<String> userNames = inputView.getUserNames();
+            int stepNum = inputView.getStepNum();
+
+            ladder = new Ladder(stepNum, userNames);
+
+            System.out.println(outputView.getStringLadder(ladder));
```

Comment:
```suggestion
           outputView.printResult(ladder);
```
View 가 있으니 이렇게 하면 어때요?